scripts/__input_initialize/__input_initialize.gml

```diff
@@ -652,6 +676,11 @@ function __input_initialize()
         }
                 
         __input_steam_type_set("unknown", "UnknownNonSteamController", "Controller");
+        
+        global.__input_steam_trigger_mode[$ string(INPUT_TRIGGER_EFFECT.TYPE_OFF)]       = steam_input_sce_pad_trigger_effect_mode_off;
+        global.__input_steam_trigger_mode[$ string(INPUT_TRIGGER_EFFECT.TYPE_FEEDBACK)]  = steam_input_sce_pad_trigger_effect_mode_feedback;
+        global.__input_steam_trigger_mode[$ string(INPUT_TRIGGER_EFFECT.TYPE_WEAPON)]    = steam_input_sce_pad_trigger_effect_mode_weapon;
+        global.__input_steam_trigger_mode[$ string(INPUT_TRIGGER_EFFECT.TYPE_VIBRATION)] = steam_input_sce_pad_trigger_effect_mode_vibration;
```

Comment:
I thhhink these relationships can be implemented as static variables in the effect classes

Reply:
Or even have a `__apply_steam()` function for the classes

Reply:
`steam_input_sce_pad_trigger_effect_mode_*` constants can't be used outside of an `global.__input_using_steamworks` case. Not going to change this myself but feel free to if you see a good solution.